.github/workflows/ci.yml

```diff
@@ -165,3 +166,14 @@ jobs:
       - name: Push Multi-Platform Package to Upbound
         if: env.XPKG_ACCESS_ID != ''
         run: "./crossplane --verbose xpkg push --package-files $(echo *.xpkg|tr ' ' ,) ${{ env.XPKG }}:${{ env.XPKG_VERSION }}"
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3.3.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push Multi-Platform Package to GHCR
+        if: env.XPKG_ACCESS_ID != ''
```

Comment:
I just happened to be copying this PR for https://github.com/crossplane-contrib/function-python and wondered why it wasn't pushing to GHCR. It was because XPKG_ACCESS_ID wasn't set. Is there a reason to gate GHCR pushes on this environment variable? I can't think of one - the Upbound push gates on it because it's a required credential for that repo.